.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+// Coloque su configuración en este archivo para sobrescribir la configuración predeterminada y de usuario.
```

Comment:
I think is better remove this file because is a Visual Studio Code file configuration. What do you think?